core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/PersonDetailViewModel.kt

```diff
@@ -60,5 +64,10 @@ class PersonDetailViewModel(
         }
     }
 
+    fun getGender(gender: Int = 0): String {
```

Comment:
I think this should be removed. Is anything still using it? It does not follow the design patterns.

Reply:
no, I will remove that